app/data_model/submitted_responses.py

```diff
@@ -0,0 +1,81 @@
+import boto3
+from botocore.exceptions import BotoCoreError, ConnectionError as BotoConnectionError
+from app.settings import EQ_ENVIRONMENT_PREFIX, EQ_SUBMISSION_VALIDITY, EQ_DYNAMODB_ENDPOINT
+from structlog import get_logger
+
+logger = get_logger()
+
+
+class SubmittedResponses(object):
+
+    def __init__(self):
+        if EQ_DYNAMODB_ENDPOINT:
+            dynamoDB = boto3.resource('dynamodb', endpoint_url=EQ_DYNAMODB_ENDPOINT)
+        else:
+            dynamoDB = boto3.resource('dynamodb')
+
+        tableName = EQ_ENVIRONMENT_PREFIX + '-submitted-responses'
+        self.table = dynamoDB.Table(tableName)
+
+    def insert_item(self, tx_id, answers, metadata, submitted, time_to_live):
+        """Insert an item to table"""
+
+        if not time_to_live or time_to_live > EQ_SUBMISSION_VALIDITY:
+            time_to_live = EQ_SUBMISSION_VALIDITY
+
+        valid_until = int(submitted + time_to_live)
+
+        answers = self.clean_empty(answers)
+        metadata = self.clean_empty(metadata)
+
+        item = {
+            'tx_id': tx_id,
+            'answers': answers,
+            'metadata': metadata,
+            'valid_until': valid_until
+        }
+
+        try:
+            response = self.table.put_item(Item=item)['ResponseMetadata']['HTTPStatusCode']
+            if response == 200:
+                return True
+            else:
+                raise BotoConnectionError()
+        except (BotoCoreError, BotoConnectionError):
+            logger.bind(tx_id=tx_id)
```

Comment:
Is `tx_id` not already bound? if not we should do it towards the beginning of the request